### ISSUE TITLE
Add Feature to Display All Tickets from the Same Customer for Each Contact

### ISSUE THREAD

**Issue by monolithonadmin** [frappe/helpdesk#1177]
I propose adding a feature to Frappe Helpdesk that allows the display of all tickets on the customer portal associated with a customer for each contact, irrespective of the ticket creator. This enhancement would streamline collaboration and improve customer support efficiency.

Currently, in the Frappe Helpdesk, it seems that only administrators have the capability to list all tickets (at the "backend" side) from multiple contacts within a single customer.

**Comment by ssiyad** [comment]
Check out `Open Tickets` in `Contact Details` tab in an individual ticket's page

**Comment by monolithonadmin** [comment]
Thank you, but that view is for staff member on /helpdesk URL. (It does not work btw). 
But Im talking about customer view on helpdesk/my-tickets path. 

Thank you

**Comment by RitvikSardana** [comment]
Hey @monolithonadmin , so basically what you are saying is, every ticket which is raised by the "contact" of a "company" should be visibles to all the contacts within that company?

**Comment by monolithonadmin** [comment]
Yes, it's correct @RitvikSardana. In real-life situations, it's common for multiple people from the same company to need access to the same tickets.

**Comment by RitvikSardana** [comment]
This is possible @backendoadmin , if you link a contact with customer, this will be made possible.

<img width="585" alt="Image" src="https://github.com/user-attachments/assets/cdfa1bf7-899f-43bb-8401-291e405fcda4" />
